packages/v4/src/content/design-guidelines/components/tooltip/tooltip.md

```diff
@@ -2,51 +2,44 @@
+<img src="./img/kebab-tooltip.png" alt="kebab icon with a “More options” tooltip label" width="582"/>  
 
-**Guidelines**
-* Keep your tooltips clear and concise. Use the fewest number of words you can without sacrificing meaning.
-* If the tooltip is a full sentence, end it with a period.
-* If information is needed for a user to complete a task (like a password character requirement), don’t hide it in a tooltip. Display it on the page instead.
-* Tooltips should provide new and valuable information. Never use a tooltip to repeat information already available in the UI.
-* Don’t use tooltips with question-circle icons to present contextual information in forms and other areas. Instead, use a popover.
+- Provide additional information on a data point or element in a chart or table.   
 
-**Common use cases**
-* On icons
-* In charts
+<img src="./img/bar-chart.png" alt='bar chart with a tooltip for Asia Pacific' width='306'/> 
 
-**Accessibility**
+- Define new or unfamiliar UI elements that aren’t described directly in the user interface.   
 
-Every time a user with a screen reader tabs into a field with a tooltip, the tooltip will be read out to them. 
+<img src="./img/disabled-edit-button.png" alt= 'disabled edit button with a tooltip about access' width='306'/>
 
-### When to use a tooltip versus a popover
+## Tooltips versus popovers
+Both tooltips and [popovers](/components/popover/design-guidelines) provide more information in context for users. However, they’re different in a few ways:
 
-Both tooltips and popovers allow users to get more information in context. However, they differ in two ways:
- 
- 1. Tooltips are used for identification purposes, while popovers are used for added description or information in context. 
- 2. Tooltips appear on hover, while popovers appear on click
+- Tooltips are used for identification purposes, while popovers are used for added description or information in context. 
+- Tooltips contain short descriptions or labels, while popovers contain longer descriptions, formatted text, and optional images or links.
+- Tooltips appear on hover, while popovers appear on click.   
+
+## Content
+- Keep your tooltips clear and concise. Use the fewest number of words without sacrificing meaning.
+- If the tooltip is a full sentence, include a period at the end. If the tooltip is a fragment, do not include a period at the end.
+- Use tooltips for additional information. Don’t use tooltips for information critical to a user completing a task, such as password character requirements.  
+- Tooltips should provide new and valuable information. Never use a tooltip to repeat information already available in the UI.
+- Don’t use tooltips with question-circle icons to present contextual information in forms and other areas. Instead, use a popover.  
```

Comment:
Let's link to the popover guidelines when you say "instead, use a popover". So just replace with: [popover](/components/popover/design-guidelines)
^ tried to write out the full code but it linked it instead haha

Reply:
Added link